examples/internal/server/a_bit_of_everything.go

```diff
@@ -28,6 +28,8 @@ import (
 var uuidgen = fastuuid.MustNewGenerator()
 
 type _ABitOfEverythingServer struct {
+	examples.UnimplementedABitOfEverythingServiceServer
+	examples.UnimplementedStreamServiceServer
```

Comment:
These will eventually be required by `protoc-gen-go-grpc`, so I'm adding them now